src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/UpdateHandler.java

```diff
@@ -0,0 +1,39 @@
+package com.minecolonies.coremod.colony.requestsystem.management.handlers;
+
+import com.minecolonies.coremod.colony.requestsystem.management.IStandardRequestManager;
+import com.minecolonies.coremod.colony.requestsystem.management.handlers.update.IUpdateStep;
+import com.minecolonies.coremod.colony.requestsystem.management.handlers.update.implementation.CraftingUpdate;
+import com.minecolonies.coremod.colony.requestsystem.management.handlers.update.implementation.InitialUpdate;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.SortedMap;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.SortedMap'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)